stan/math/prim/scal/prob/bernoulli_logit_rng.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef STAN_MATH_PRIM_SCAL_PROB_BERNOULLI_LOGIT_RNG_HPP
+#define STAN_MATH_PRIM_SCAL_PROB_BERNOULLI_LOGIT_RNG_HPP
+
+#include <boost/random/bernoulli_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/inv_logit.hpp>
+#include <stan/math/prim/scal/fun/log1m.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+
+namespace stan {
+  namespace math {
+
+    template <class RNG>
+    inline int
+    bernoulli_logit_rng(const double t,
```

Comment:
Don't need const for primitive types as it doesn't change the external API.  It is usually good style in the body of the function to treat the arguments as constant, but that doesn't need to be exposed to the outside world.